open_dread_rando/files/schema.json

```diff
@@ -296,6 +296,14 @@
                 }
             },
             "additionalProperties": false
+        },
+        "entity_removal": {
+            "type": "array",
+            "description": "List of entities that should be removed.",
+            "items": {
+                "$ref": "#/$defs/actor_reference_with_layer"
+            },
+            "default": []
```

Comment:
I think I'd prefer we use specific settings in `game_patches` rather than the simple list - some settings may do more than just remove entities and I'd rather we have all the behaviour for a given patch in the same place

Reply:
I liked doing this because I have a list of EventNodes in Randovania of which you'll start with the resources, and the same list gives me the elements to remove.

But we can make flags, no problem.